src/pages/en/guides/markdown-content.md

```diff
@@ -178,32 +177,80 @@ You can import components into your Markdown file with `setup` and use them alon
 ```markdown
 ---
 layout: ../layouts/BaseLayout.astro
-setup: | 
+setup: |
   import Author from '../../components/Author.astro'
   import Biography from '../components/Biography.jsx'
 author: Leon
 ---
+
 <Author name={frontmatter.author}/>
 <Biography client:visible>
   {frontmatter.author} lives in Toronto, Canada and enjoys photography.
 </Biography>
 ```
 
-<!-- 
-
 ## Importing Markdown
 
-https://github.com/withastro/rfcs/pull/147
+You can import Markdown files directly into your Astro files! You can import one specific page with `import` or multiple with `Astro.glob()`
+
+```astro
+---
+// You can use `import ... from`
+import * as greatPost from '../pages/post/great-post.md';
+
+// Or even `await import()`
+const aboutUs = await import('../pages/internal/about-us.md');
```

Comment:
Is there an advantage to using one of these over the other? Even if there are two ways to do this, if they’re identical, maybe it’s a good idea just to document one for simplicity? Maybe the `await import` version as it is more similar to how `Astro.glob` is used?

Reply:
cc @FredKSchott

Reply:
Chris' instinct is a good one. I'd also be interested to know if we have a preference to nudge people towards, and suspect that choosing the form that more closely resembles Astro.glob is a good choice for consistency.

But, I don't think that blocks this PR, and have no problems publishing this as-is and revisiting later if want to.

Reply:
I'd advise against using await if plain ESM suits your needs. I only use await import when rendering dynamically inside the Astro templating itself.

Reply:
I've just updated that bit, what do you think?

Reply:
I like it!

Reply:
personal preference -- I usually recommend this format instead:

```
---
import {Content as PromoBanner} from '../components/promoBanner.md';
--- 
{promoActive() && <PromoBanner/>}
```

As for documenting static vs. dynamic imports, we should push users towards static when we can, but it can still be nice to know that dynamic imports are supported. Maybe something more like this?

```diff
// Import some markdown. Dynamic import() is also supported!
import * as greatPost from '../pages/post/great-post.md';
- // Or even `await import()`
- const aboutUs = await import('../pages/internal/about-us.md');
```

but fwiw I like how you have it already also, so I'm +1 either way / leave the final wording up to you.

Reply:
So... back to the original, @JuanM04 ? 😅   Either your latest version, or the original/Fred's is fine with me!

Reply:
~~Dealer's~~ Juan's choice!

Reply:
> Either your latest version, or the original/Fred's is fine with me!

¿Por qué no los dos?

Reply:
muí caliente! 🔥 

(If there's any way to get a little more spacing at the bottom between the code sample and the next list item, it would look a little better.)

Reply:
I don't know how to do that, so I moved the items around [bda5eaa](https://github.com/withastro/docs/pull/405/commits/bda5eaaa39d4c8cf9fadd846bbb241a88f72c0f9)

Reply:
I love it even more! Winner?

Reply:
Looking gooooood!!!

Reply:
LGTM!